### ISSUE TITLE
Cleaning up cogito monorepo build system

### ISSUE THREAD

**Issue by marcinczenko** [philips-software/cogito#44]
We have enough experience now, to clean this is up a bit. For all our packages, we now are generating `commonjs`, `es modules`, and `umd`. For most of the packages we even use `umd` version for `pkg.main`. So when we bundle our app, webpack is using `es modules`, when we run the tests with jest, it uses `umd` (jest needs `commonjs` compatible build). Our initial proposition was to support using our cogito packages directly in the browser with `<script>` tag. But, at the current stage it is hard to imagine that this will ever be really used by anyone and this puts lots of burden to maintain `rollup` and its configuration. Additionally, using `rollup` made us doing some unclear things with `peer-dependencies`. I think that it is save to drop support for `umd` or any other (e.g. `iife`) `<script>` usage scenario for the moment. This will simplify our workspaces a lot. I initially thought that `iife` will be nice to still have for zero-configuration bundlers like *parcel* but this is not needed - *parcel* can consume both `commonjs` and `es module` syntax.

Additionally, I suggest we remove duplication from the workspaces for the `devDependencies`. Things like:

```
"@babel/cli": "^7.0.0",
"@babel/core": "^7.0.0",
"@babel/plugin-proposal-class-properties": "^7.0.0",
"@babel/plugin-proposal-object-rest-spread": "^7.0.0",
"@babel/preset-env": "^7.0.0",
"@babel/preset-react": "^7.0.0",
"babel-core": "^7.0.0-0",
"babel-eslint": "9.0.0",
"babel-jest": "^23.6.0",
```

should be kept only at the top-level of monorepo and only when diverging inside of workspaces.

So, to summarise:

1. get rid of rollup
2. remove duplication from workspaces and move it to the top of the monorepo
3. clean up the `peer-dependencies`.

After this, cogito build system will be way easier to maintain.

**Comment by marcinczenko** [comment]
This one is handled in #46.
